components/public-api/gitpod/experimental/v1/workspaces.proto

```diff
@@ -25,6 +25,9 @@ service WorkspacesService {
     // CreateAndStartWorkspace creates a new workspace and starts it.
     rpc CreateAndStartWorkspace(CreateAndStartWorkspaceRequest) returns (CreateAndStartWorkspaceResponse) {}
 
+    // StartWorkspace starts an existing workspace.
+    rpc StartWorkspace(StartWorkspaceRequest) returns (StartWorkspaceResponse) {}
```

Comment:
So technically, `StartWorkspace` actually makes very little sense from the outside.

Either you want to `CreateWorkspace` when you don't have an existing one, or you want to `RestartWorkspace` because you've got one which terminated.

`StartWorkspace` is effectively an internal implementation, because we separated the process into Create and then Start

Would it make sense to make this `RestartWorkspace` which already communicates what you need to be able to do the action?

Reply:
Won't restart will imply stop + start?

i.e docker engine for containers has:
- create
- start
- stop
- restart as a shortcut for stop + start

Reply:
I also don't think 'start' is ambiguous.